web/src/features/charts/ChartTitle.tsx

```diff
@@ -21,9 +21,9 @@ export function ChartTitle({ translationKey, badgeText = undefined, icon, unit }
   return (
     <div className="flex flex-col pb-0.5">
       <div className="flex flex-row justify-between pt-4">
-        <div className="flex content-center gap-1.5">
+        <div className="flex content-center items-center gap-1.5">
           {icon && <div className="w-5">{icon}</div>}
-          <h3 className="text-md font-bold">{t(`${translationKey}.${timeAverage}`)}</h3>
+          <h2>{t(`${translationKey}.${timeAverage}`)}</h2>
```

Comment:
What are the thoughts on making this a `h2`? In Figma it's using text-sm (at least in the file I looked at, sorry if you have another reference)
![Screenshot 2024-07-01 at 15 03 56](https://github.com/electricitymaps/electricitymaps-contrib/assets/3296643/5d0166c4-e3b8-4a6f-8c15-2b2302f43c56)

Reply:
Sorry, Figma is all over the place. The reference is [in the design system file for now](https://www.figma.com/design/BhpQbMBpicEYZeQ9aKKW7j/%E2%9A%A1%EF%B8%8F-The-Grid---Design-System-%E2%9A%A1%EF%B8%8F?node-id=959-12162&t=zREq1v94EsHwEdsT-1). It is an H2 there 🥳